datadog/__init__.py

```diff
@@ -14,6 +14,7 @@
 import logging
 import os
 import os.path
+from typing import Any, List, Optional
```

Comment:
just a heads up. this is only part of the std lib for python 3.5 and greater. so out of the  box `pip install datadog` for versions earlier than that will run successfully but unless typing is also installed the initial import will fail.

Reply:
happy to submit an issue/fix if needed

Reply:
In [setup.py](https://github.com/DataDog/datadogpy/pull/565/files#diff-2eeaed663bd0d25b7e608891384b7298), the `typing` package is included in `install_requires` if the user is running Python < 3.5.